SmartDeviceLink/SDLHMICapabilities.h

```diff
@@ -29,6 +29,13 @@ NS_ASSUME_NONNULL_BEGIN
  */
 @property (nullable, copy, nonatomic) NSNumber<SDLBool> *videoStreaming;
 
+/**
+ Availability of built in remote control. True: Available, False: Not Available
+
+ Boolean value. Optional.
```

Comment:
Mark when it was made available as well, please (other parameters and the class itself should also have that info).